### ISSUE TITLE
Dependencies too old

### ISSUE THREAD

**Issue by nerdnils** [opensludge/opensludge#9]
Hi there, does anybody still read this or is this abandoned since 2016? Latest release is even 2014…
I got a game written in sludge I really need to get running again. And this does not compile anymore because it got automake1.14 and others as dependencies which is way too old to get it to work again on modern systems.

Can this be updated?

Greetings

**Comment by dos1** [comment]
I have a port to SDL2 hanging somewhere, but I'd have to get some confirmation that this work won't get stalled in PR queue forever before taking effort to upstream it. Seems like nobody is maintaining it right now, so there's nobody to accept PRs and make releases.

@tobihan, how about giving some access to the repo and the website to @tobiatesan, who seems to be somebody with the best internal Sludge knowledge out there that might still be willing to do some light maintenance, or maybe even @nerdnils or me? @tobiatesan, would you even be interested?

**Comment by tobihan** [comment]
Yes, I could give access if someone is interested in maintaining sludge.

@nerdnils Could you be more specific what does not work with automake > 1.14?

**Comment by nerdnils** [comment]
@tobihan the PKGBUILD in aur said so. But it turned out, it was wrong. But dos1 even seems to work on it, porting it. So it would be great that this could come back to life.

**Comment by dos1** [comment]
@tobihan Okay, so as nobody else jumps in, maybe could you give access to the repo? I sure won't have enough time to actively work on SLUDGE, but the contribution traffic here seems manageable, and I definitely want to sit down make new builds of SLUDGE someday, so why not make them official :)

**Comment by dos1** [comment]
@tobihan Thanks, I've joined the organization - however, I don't have write access to the repo yet :(

**Comment by dos1** [comment]
Okay, I've got write permissions working now :)

So, just that you all know: I'd like to get proper builds of current master for every platform and release them as SLUDGE 2.2.2.

Then, after switching to SDL2 (and possibly getting rid of some other depreciated dependencies as well - maaaaaybe switching to GTK+ 3 too), I'd like to release SLUDGE 2.3.

...and that would be it, with new point releases done occasionally when some external contribution happens.

**Comment by tobihan** [comment]
I would like to release SLUDGE 2.2.2 to get the various fixes out. Does anyone want to get in any other changes? If I don't hear anything I'll do the release in a few days.

**Comment by dos1** [comment]
I have spent some time with SLUDGE today. I've pushed some small fixes and opened #16 - I think that's all from me right now.
